MainRepository/Program.cs

```diff
@@ -1,7 +1,26 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Program to convert the decimal number to binary and hexadecimal number.
+Console.Write ("Enter the number: ");
+List<int> binary = new ();
+List<char> hexadecimal = new ();
+List<char> hexValues = new () { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F' };

```

Comment:
Instead of using List<char>, you could use string. You can use indexing operation with string.